### PR TITLE
ENG-9437-ycsb-add-ratio

### PR DESCRIPTION
**Pull request by prosegay**
xin, please take a look

### CODE REVIEW

File: tests/test_apps/ycsb/ycsb_scaler.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+            print "ERROR voltdb.rowsize must be > 0"
+            sys.exit(1)
+
+        if voltdb_scalefactor <= 0:
+            print "ERROR voltdb.rowsize must be > 0"
+            sys.exit(1)
+
+        # get phys memory installed on one server
+        # assume servers are symmetrical
+        physical_memory = int(call_sqlcmd("exec @Statistics MEMORY 0", servers[0]).split('"')[-4]) * 1024
```
Comment (xinjiacs) [inline tests/test_apps/ycsb/ycsb_scaler.py]: @prosegay Does this assume the servers have the same physical memory as servers[0]?

File: tests/test_apps/ycsb/ycsb_scaler.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+        except:
+            pass
+    raise Exception("property '%s' not found" % prop)
+
+
+if __name__ == "__main__":
+
+    voltdb_rowsize = None
+    voltdb_scalefactor = None
+    recordcount = None
```
Comment (xinjiacs) [inline tests/test_apps/ycsb/ycsb_scaler.py]: recordcount is assigned later but not used anywhere else, maybe remove it?

File: tests/test_apps/ycsb/ycsb_scaler.py [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+
+    pristine = dict()   # properties frozen set copy
+    properties = dict() # properties working dir
+
+    for p in range(0, len(sys.argv)):
```
Comment (xinjiacs) [inline tests/test_apps/ycsb/ycsb_scaler.py]: Maybe it's worth to add an example sys.argv, then it's easier to read the code with that exmaple.

**Comment by xinjiacs** [comment]
That's all my comments, thanks.

File: tests/test_apps/ycsb/ycsb_scaler.py (hunk 1 above)
Comment (prosegay) [inline tests/test_apps/ycsb/ycsb_scaler.py]: yes, is that ok?

File: tests/test_apps/ycsb/ycsb_scaler.py (hunk 2 above)
Comment (prosegay) [inline tests/test_apps/ycsb/ycsb_scaler.py]: its used at line 100

Comment (xinjiacs) [inline tests/test_apps/ycsb/ycsb_scaler.py]: I know that, but is's just got assigned without using it anywhere else.

File: tests/test_apps/ycsb/ycsb_scaler.py (hunk 1 above)
Comment (xinjiacs) [inline tests/test_apps/ycsb/ycsb_scaler.py]: I guess it's fine for the benchmark now.

**Pull request closed without merging**